G1ANT.Addon.OCR.Tesseract/G1ANT.Addon.OCR.Tesseract/OcrOfflineHelper.cs

```diff
@@ -92,5 +99,21 @@ public static void UnpackNeededAssemblies()
                 }
             }
         }
+
+        private static void UnpackTesseract()
+        {
+            var executingPath = Path.GetDirectoryName(System.Reflection.Assembly.GetExecutingAssembly().Location);
```

Comment:
do you think that `GetExecutingAssembly().Location` is correct place to copy files to?
G1ANT.Robot.exe is standalone executable and can be possibly run from Desktop.

Reply:
dll is not a part of robot exe file (huge size), it could be placed for eg. in MyDocuments/G1ANT.Robot folder